Documentation/README.md

```diff
@@ -145,7 +145,7 @@ In your new project add a new NuGet.config to point to the Microsoft.NETCore.App
   <!-- The command-line doesn't need it, but the IDE does.                    -->
   <packageSources>
     <clear/>
-    <add key="local feed" value="C:\repos\source-build\bin\obj\x64\Release\blob-feed\packages" />
+    <add key="local feed" value="E:\repos\source-build\bin\x64\Release\runtime" />
```

Comment:
Doesn't the core-setup.proj change only copy the installers/tarballs (assets) to this location, not the nupkgs? I'd think the packages dir would still be the right nuget source. The earlier changed line in this file might also be better if it described both places.